identity/handler_test.go

```diff
@@ -503,6 +519,112 @@ func TestHandler(t *testing.T) {
 			}
 		})
 
+		t.Run("case=PATCH update of state should update state changed at timestamp", func(t *testing.T) {
```

Comment:
We require a test for PATCH validating that an identities `credentials` cannot be updated through this method.

Reply:
Right, I added the test.